bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/RifLayout.java

```diff
@@ -301,6 +301,8 @@ public String getJavaFieldName() {
 
 		DATE,
 
-		NUM;
+		NUM,
+		
+		TIME;
```

Comment:
Is this just a time, or a combined date-time-stamp? If the latter, I'd call it "TIMESTAMP" to be clear.